stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/MoshiChatApi.kt

```diff
@@ -83,8 +83,8 @@ internal class MoshiChatApi(
 
     val logger = ChatLogger.get("MoshiChatApi")
 
-    lateinit var userId: String
-    lateinit var connectionId: String
+    private var userId: String = ""
+    private var connectionId: String = ""
```

Comment:
Not to crash from notification as we don't connect to the WS and therefore we don't have `connectionId`